Final/Final.tsx

```diff
@@ -1,7 +1,8 @@
 import React from "react";
 import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
 import { EventsStack } from "./EventsStack";
-import { AttendScreen } from "./AttendScreen";
+import AttendScreen from "./AttendScreen";
+import Ionicon from '@expo/vector-icons/Ionicons';
 
 const Tab = createBottomTabNavigator();
```

Comment:
The Tab navigator should have a type associated with it so your Screen can tell if the component is valid or not.